slack_connector/main.py

```diff
@@ -51,14 +74,19 @@ def get_messages(api_token,channel_id):
 
             try:
 
-                result = client.conversations_history(channel=c)
+                result = client.conversations_history(channel = c)
                 conversation_history = result["messages"]
                 df_conversation_history = pd.DataFrame(conversation_history)
```

Comment:
Probably it's best moving this DataFrame creation to line after the conversation_history check statement?